octoprint_mrbeamdoc/test___init__.py

```diff
@@ -0,0 +1,114 @@
+from unittest import TestCase
+
+from mock import patch
+
+from octoprint_mrbeamdoc import get_doc_path, MrBeamDocAvailable, SupportedLanguage, MrBeamDocNotFoundException, \
+    MrBeamModel, MrBeamDocType
+
+
+class Test(TestCase):
+    def test_happy_path_german(self):
+        mrbeamdoc = get_doc_path(MrBeamDocType.QUICKSTART_GUIDE, MrBeamModel.MRBEAM2, SupportedLanguage.GERMAN)
```

Comment:
suggestion of test case name format to follow:
test_get_doc_path_with_german_should_succeed

explanation:
test_ --> prefix
get_doc_path --> unit under test
with(out) .... --> parameters/conditions for the test
should(Not) ... --> expected behavior 

it would lengthen the test name but it makes the tests more readable and reduce the need for extra documentation